config/locales/en/globals.yml

```diff
@@ -0,0 +1,3 @@
+en:
```

Comment:
I'm keen to avoid a file called 'globals', as it could contain absolutely anything. We can take the approach for 'states' and 'validations' and call it 'documents'.